### ISSUE TITLE
Refresh API return error message

### ISSUE THREAD

**Issue by manojsitapara** [open-wa/wa-automate-nodejs#1965]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### Current Behavior

When invoking refresh API end point, I am getting following error

`Evaluation failed: ReferenceError: WAPI is not defined\n    at __puppeteer_evaluation_script__:1:117837\n    at __puppeteer_evaluation_script__:1:400050`

![image](https://user-images.githubusercontent.com/12126671/127745233-26ad74c2-8455-4476-b856-724d51c441d2.png)

### Mode

My own code

### Environment

```markdown
- OS: Ubuntu 20.04
```

**Comment by jaykon-w** [comment]
same error here

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.13.3](https://github.com/open-wa/wa-automate-nodejs/compare/4.13.2...4.13.3) (2021-08-06)

- Release 4.13.3 [`43a9ddb`](https://github.com/open-wa/wa-automate-nodejs/commit/43a9ddbe0f34a5bdc71177140536c50cee2b66da)
- 🐛 fix: `refresh` now works as expected #1965 [`acca9d3`](https://github.com/open-wa/wa-automate-nodejs/commit/acca9d3a7b9252d76b8d686f7858eb34b800a9aa)
- 🥚 patches update [`3bd5002`](https://github.com/open-wa/wa-automate-nodejs/commit/3bd50028a51b84d1b1601cbdad0d82eca4c21724)
- fix `multi-device enabled` message [`f05e61b`](https://github.com/open-wa/wa-automate-nodejs/commit/f05e61b07072ec7281cbd8b025a4c116a1bead85)

**Comment by smashah** [comment]
Should be working now thanks
